records/tools/uvision.uvproj.tmpl

```diff
@@ -72,6 +70,7 @@
             <UserProg2Name></UserProg2Name>
             <UserProg1Dos16Mode>0</UserProg1Dos16Mode>
             <UserProg2Dos16Mode>0</UserProg2Dos16Mode>
+            <nStopB1X>2</nStopB1X>
```

Comment:
Bummer. Oh well, hopefully most people use v5 and benefit from this. I assume the added elements will just be ignored by v4. Can you confirm? (I don't have it installed)